samples/java-eventsourced-shopping-cart/src/main/resources/application.conf

```
@@ -0,0 +1,4 @@
+
+# //tag::snapshot-every[]
+akkaserverless.event-sourced-entity.event-sourced-entity.snapshot-every = 100
```

Double `event-sourced-entity`

```suggestion
akkaserverless.event-sourced-entity.snapshot-every = 100
```